libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -150,7 +150,7 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @Param: OPTIONS
     // @DisplayName: Battery monitor options
     // @Description: This sets options to change the behaviour of the battery monitor
-    // @Bitmask: 0:Ignore DroneCAN SoC, 1:MPPT reports input voltage and current, 2:MPPT Powered off when disarmed, 3:MPPT Powered on when armed, 4:MPPT Powered off at boot, 5:MPPT Powered on at boot, 6:Send resistance compensated voltage to GCS
+    // @Bitmask: 0:Ignore DroneCAN SoC, 1:MPPT reports input voltage and current, 2:MPPT Powered off when disarmed, 3:MPPT Powered on when armed, 4:MPPT Powered off at boot, 5:MPPT Powered on at boot, 6:Send resistance compensated voltage to GCS, 7:Send resistence compensated voltage to telemetry
```

Comment:
Could we re-use the GCS option and widen its scope? "Send resistance compensated voltage to GCS and telemetry"

I can't see any case where you would want them to be different.

Reply:
That was my initial approach, though I didn't want to presume to do so. However, GCS and telem are different avenues. Maybe we shouldn't assume what someone would want?

Reply:
We also shouldn't add features unless anyone actually wants them. I think its safe to assume in this case. If someone turns up and wants different options for each we can add the new option at that point.

Reply:
I wanted it https://github.com/ArduPilot/ardupilot/issues/24935 and I was being proactive in implementing it myself. I think a lot of people would find this helpful even if they didn't go out of their way to request it.

Reply:
To be clear, I think the feature itself worth having and thank you for putting the effort to implement it.

What I don't think we need is the two options. I don't think you want to report a different voltage to the GCS vs  RC telemetry?

Reply:
I agree. As I said that was my initial approach. Just wanted to open it up for thought, rather than assuming.
I'll make that change. Would it be appropriate then to rename GCS_Resting_Voltage to Telem_Resting_Voltage? Seems that terminology would apply to all.

Reply:
Re naming, I also think that "telem" is a better name than "GCS".  Sadly we use "GCS" to mean telemetry all over the place so we have a conflict between naming things properly and maintaining consistency.  I guess I'd tend towards leaving the internal methods as they but making the param description slightly more clear without making it too long.  The final option is to "fix it in documentation" which means update the wiki.

Reply:
In that case why not get rid of the gcs_voltage() in place of telem_voltage()? Be a force for change and all that.
@IamPete1 suggestion is only 2 additional words: "and telemetry", or change to "Use sag compensated voltage for telemetry". The latter making more sense to me because I think that's the terminology most would identify with (not resting voltage) and that's the terminology BATT_FS_VOLTSRC uses: Sag compensated voltage. Also the verbiage, imo, is more natural and clear; "send...to..." makes it sound like your sending something additional.

Reply:
I've removed GCS_Resting_Voltage in favor of just Telem_Resting_Voltage, and changed the option desc to "Use sag compensated voltage for telemetry".